backend/hook/pg_lib.go

```diff
@@ -12,26 +12,26 @@ import (
 )
 
 type pgLibImpl struct {
-	db       *gorm.DB
-	modelMap map[string]database.Model
+	db       map[string]*gorm.DB
+	modelMap map[string]map[string]database.Model
 }
 
 // NewPGLib dblib implement by postgres
-func NewPGLib(db *gorm.DB, modelMap map[string]database.Model) DBLib {
+func NewPGLib(db map[string]*gorm.DB, modelMap map[string]map[string]database.Model) DBLib {
 	return &pgLibImpl{
 		db:       db,
 		modelMap: modelMap,
 	}
 }
 
-func (s *pgLibImpl) First(tableName string, condition string) (map[interface{}]interface{}, error) {
-	model, ok := s.modelMap[tableName]
+func (s *pgLibImpl) First(dbName string, tableName string, condition string) (map[interface{}]interface{}, error) {
```

Comment:
We should check DB name exist and return error in all (i will make panic if dbName is not exist), check all